CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Devel
 
+- Removed lint, removed old baserepo standalone code, and added travis config.
```

Comment:
Want to split this into two things? (lint/old code & travis)

I'm no stickler for "one change per PR"